capture/config.default.json

```diff
@@ -2,20 +2,23 @@
 	"viewports" : [
 		{
 		 "name": "phone",
-		 "viewport": {"width": 320, "height": 480}
+		 "width": 320,
+     "height": 480
```

Comment:
Inconsistent indentation. I don't know about you guys, but I use 2 spaces instead of a tab for all javascript. Configured my editor to do it that way when I hit the tab key, and to auto indent on new line.